pkg/controllers/management/clusterstats/statsaggregator.go

```diff
@@ -220,3 +220,13 @@ func (s *StatsAggregator) machineChanged(key string, machine *v3.Node) (runtime.
 	}
 	return nil, nil
 }
+
+func isWorkerNode(machine *v3.Node) bool {
+	if machine.Spec.Worker {
+		return true
+	}
+	if !machine.Spec.Worker && !machine.Spec.ControlPlane && !machine.Spec.Etcd {
```

Comment:
First part seems redundant but thats ok I guess. So we aren't checking labels/annotations at all anymore like original PR ? Is it possible box has a label/annotation of control plane ?

Reply:
I see those linked PR's now. I guess this works. Might be nice to explain with comment here why we assume its worker.

Reply:
it makes sense not to re-check the labels/annotations and rely on the Spec as a source of truth - unless there is a reason that I am missing the nodesync stuff wouldn't have occurred for a machine

Reply:
this method was removed